src/index.json

```diff
@@ -737,12 +737,11 @@
         ],
         "dms-preview": [
             {
-                "downloadUrl": "https://azcliorcas.blob.core.windows.net/azclipath/dms_preview-0.8.1-py2.py3-none-any.whl",
-                "filename": "dms_preview-0.8.1-py2.py3-none-any.whl",
```

Comment:
You should keep the old entry so there is a version that can be installed on older versions of the CLI.

Reply:
Either you can keep the old entry. Or you can keep "azext.minCliCoreVersion": "2.0.43" and remove max cli core version. As the new code with min_profile can work with all core versions

Reply:
I would go with removing max and keeping the original min.

Reply:
Updated the version the original version.